src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1315,6 +1315,31 @@ bool AdaptDispatch::SetTopBottomScrollingMargins(const size_t topMargin,
     return _DoSetTopBottomScrollingMargins(topMargin, bottomMargin) && CursorPosition(1, 1);
 }
 
+// Routine Description:
+// - BEL - Rings the warning bell.
+//    Causes the terminal to emit an audible tone of brief duration.
+// Arguments:
+// - None
+// Return Value:
+// - True if handled successfully. False otherwise.
+bool AdaptDispatch::WarningBell()
+{
+    return _pConApi->PrivateWarningBell();
+}
+
+// Routine Description:
+// - CR - Performs a carriage return.
+//    Moves the cursor to the leftmost column.
+// Arguments:
+// - None
+// Return Value:
+// - True if handled successfully. False otherwise.
+bool AdaptDispatch::CarriageReturn()
+{
+    // TODO: This will need to be rewritten after PR #3628
```

Comment:
Should we make a task for that?

Reply:
(note that this is the reason I haven't yet hit the automerge button. If we should just do this _now_, let's do it now, otherwise lets file a follow up and merge this now.)

Reply:
I'll have a look at that now.